kuma/users/jinja2/socialaccount/signup.html

```diff
@@ -37,7 +37,7 @@ <h1>{{ _('Create your MDN account') }}</h1>
         </li>
       {% endif %}
       <li class="mdn-profile">
-        <img src="{{ static('auth/mdn-auth.svg') }}" width="90px" alt="{{ _('User avatar') }}" />
+        <img src="{{ account.get_avatar_url() }}" width="90px" alt="{{ _('User avatar') }}" />
```

Comment:
Perhaps I'm old but I learned back in the ancient days of decades ago that the `width` was just a number. Can you use `px` in them these days?

Reply:
Ha! weird. It will probably just be ignored. Not sure why it has `px`